src/test/java/com/philips/research/bombar/core/domain/licenses/ObligationsAnalyzerTest.java

```diff
@@ -0,0 +1,123 @@
+package com.philips.research.bombar.core.domain.licenses;
+
+import com.philips.research.bombar.core.domain.Dependency;
+import com.philips.research.bombar.core.domain.Project;
+import com.philips.research.bombar.core.domain.Relation;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ObligationsAnalyzerTest {
+    private static final String OTHER_OBLIGATION = "Other";
```

Comment:
Seems odd to first encounter the "other" obligation, and then later find the "main" obligation on line 21.

Always sort the constants to make it easier to recognise which constants are available.